sklearn/model_selection/_search.py

```diff
@@ -61,6 +62,9 @@ class ParameterGrid(object):
         useful to avoid exploring parameter combinations that make no sense
         or have no effect. See the examples below.
 
+    least_significant : str or list of str, optional
```

Comment:
Or can we have a `search_order`?

Reply:
from the decribtion, I don't understand what it means...